### DISCUSSION TITLE
Problems with sudo apt install

### DISCUSSION THREAD

**Discussion by EberhardBinder** [eugene-tarassov/vivado-risc-v#159] [Q&A] [answered]
I am using the Linux-Version and try to update the system.

If I want for example to instal gcc I get the following error message:

root@debian:~# sudo apt install gcc
sudo: unable to resolve host debian: Temporary failure in name resolution
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  binutils binutils-common binutils-riscv64-linux-gnu cpp cpp-12
  fontconfig-config fonts-dejavu-core gcc-12 libasan8 libbinutils libbrotli1
  libc-dev-bin libc-devtools libc6-dev libcc1-0 libcrypt-dev libctf-nobfd0
  libctf0 libdeflate0 libfontconfig1 libfreetype6 libgcc-12-dev libgd3
  libgomp1 libisl23 libjbig0 libjpeg62-turbo liblerc4 libmpc3 libmpfr6
  libnsl-dev libpng16-16 libtiff5 libtirpc-dev libwebp7 libx11-6 libx11-data
  libxau6 libxcb1 libxdmcp6 libxpm4 linux-libc-dev manpages manpages-dev
  rpcsvc-proto
Suggested packages:
  binutils-doc cpp-doc gcc-12-locales cpp-12-doc gcc-multilib make autoconf
  automake libtool flex bison gdb gcc-doc gcc-12-doc glibc-doc libgd-tools
  man-browser
The following NEW packages will be installed:
  binutils binutils-common binutils-riscv64-linux-gnu cpp cpp-12
  fontconfig-config fonts-dejavu-core gcc gcc-12 libasan8 libbinutils
  libbrotli1 libc-dev-bin libc-devtools libc6-dev libcc1-0 libcrypt-dev
  libctf-nobfd0 libctf0 libdeflate0 libfontconfig1 libfreetype6 libgcc-12-dev
  libgd3 libgomp1 libisl23 libjbig0 libjpeg62-turbo liblerc4 libmpc3 libmpfr6
  libnsl-dev libpng16-16 libtiff5 libtirpc-dev libwebp7 libx11-6 libx11-data
  libxau6 libxcb1 libxdmcp6 libxpm4 linux-libc-dev manpages manpages-dev
  rpcsvc-proto
0 upgraded, 46 newly installed, 0 to remove and 0 not upgraded.
Need to get 43.4 MB/44.8 MB of archives.
After this operation, 168 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Err:1 http://deb.debian.org/debian-ports unstable/main riscv64 manpages all 6.01-1
  404  Not Found [IP: 146.75.118.132 80]
Err:2 http://deb.debian.org/debian-ports unstable/main riscv64 binutils-common riscv64 2.39-8
  404  Not Found [IP: 146.75.118.132 80]
Err:3 http://deb.debian.org/debian-ports unstable/main riscv64 libbinutils riscv64 2.39-8
  404  Not Found [IP: 146.75.118.132 80]
Err:4 http://deb.debian.org/debian-ports unstable/main riscv64 libctf-nobfd0 riscv64 2.39-8
  404  Not Found [IP: 146.75.118.132 80]
Err:5 http://deb.debian.org/debian-ports unstable/main riscv64 libctf0 riscv64 2.39-8
  404  Not Found [IP: 146.75.118.132 80]
Err:6 http://deb.debian.org/debian-ports unstable/main riscv64 binutils-riscv64-linux-gnu riscv64 2.39-8
  404  Not Found [IP: 146.75.118.132 80]
Err:7 http://deb.debian.org/debian-ports unstable/main riscv64 binutils riscv64 2.39-8
  404  Not Found [IP: 146.75.118.132 80]
Err:8 http://deb.debian.org/debian-ports unstable/main riscv64 libmpfr6 riscv64 4.1.0-3
  404  Not Found [IP: 146.75.118.132 80]
Err:9 http://deb.debian.org/debian-ports unstable/main riscv64 libmpc3 riscv64 1.2.1-2
  404  Not Found [IP: 146.75.118.132 80]
Err:10 http://deb.debian.org/debian-ports unstable/main riscv64 cpp-12 riscv64 12.2.0-9
  404  Not Found [IP: 146.75.118.132 80]
Err:11 http://deb.debian.org/debian-ports unstable/main riscv64 cpp riscv64 4:12.2.0-1
  404  Not Found [IP: 146.75.118.132 80]
Err:12 http://deb.debian.org/debian-ports unstable/main riscv64 fonts-dejavu-core all 2.37-2
  404  Not Found [IP: 146.75.118.132 80]
Err:13 http://deb.debian.org/debian-ports unstable/main riscv64 fontconfig-config riscv64 2.13.1-4.5
  404  Not Found [IP: 146.75.118.132 80]
Err:14 http://deb.debian.org/debian-ports unstable/main riscv64 libcc1-0 riscv64 12.2.0-9
  404  Not Found [IP: 146.75.118.132 80]
Err:15 http://deb.debian.org/debian-ports unstable/main riscv64 libgomp1 riscv64 12.2.0-9
  404  Not Found [IP: 146.75.118.132 80]
Err:16 http://deb.debian.org/debian-ports unstable/main riscv64 libasan8 riscv64 12.2.0-9
  404  Not Found [IP: 146.75.118.132 80]
Err:17 http://deb.debian.org/debian-ports unstable/main riscv64 libgcc-12-dev riscv64 12.2.0-9
  404  Not Found [IP: 146.75.118.132 80]
Err:18 http://deb.debian.org/debian-ports unstable/main riscv64 gcc-12 riscv64 12.2.0-9
  404  Not Found [IP: 146.75.118.132 80]
Err:19 http://deb.debian.org/debian-ports unstable/main riscv64 gcc riscv64 4:12.2.0-1
  404  Not Found [IP: 146.75.118.132 80]
Err:20 http://deb.debian.org/debian-ports unstable/main riscv64 libbrotli1 riscv64 1.0.9-2+b4
  404  Not Found [IP: 146.75.118.132 80]
Err:21 http://deb.debian.org/debian-ports unstable/main riscv64 libc-dev-bin riscv64 2.36-4
  404  Not Found [IP: 146.75.118.132 80]
Err:22 http://deb.debian.org/debian-ports unstable/main riscv64 libpng16-16 riscv64 1.6.38-2
  404  Not Found [IP: 146.75.118.132 80]
Err:23 http://deb.debian.org/debian-ports unstable/main riscv64 libfreetype6 riscv64 2.12.1+dfsg-3
  404  Not Found [IP: 146.75.118.132 80]
Err:24 http://deb.debian.org/debian-ports unstable/main riscv64 libfontconfig1 riscv64 2.13.1-4.5
  404  Not Found [IP: 146.75.118.132 80]
Err:25 http://deb.debian.org/debian-ports unstable/main riscv64 libjpeg62-turbo riscv64 1:2.1.2-1+b1
  404  Not Found [IP: 146.75.118.132 80]
Err:26 http://deb.debian.org/debian-ports unstable/main riscv64 libjbig0 riscv64 2.1-6
  404  Not Found [IP: 146.75.118.132 80]
Err:27 http://deb.debian.org/debian-ports unstable/main riscv64 libwebp7 riscv64 1.2.2-2+b1
  404  Not Found [IP: 146.75.118.132 80]
Err:28 http://deb.debian.org/debian-ports unstable/main riscv64 libtiff5 riscv64 4.4.0-5
  404  Not Found [IP: 146.75.118.132 80]
Err:29 http://deb.debian.org/debian-ports unstable/main riscv64 libx11-data all 2:1.8.1-2
  404  Not Found [IP: 146.75.118.132 80]
Err:30 http://deb.debian.org/debian-ports unstable/main riscv64 libx11-6 riscv64 2:1.8.1-2
  404  Not Found [IP: 146.75.118.132 80]
Err:31 http://deb.debian.org/debian-ports unstable/main riscv64 libxpm4 riscv64 1:3.5.12-1
  404  Not Found [IP: 146.75.118.132 80]
Err:32 http://deb.debian.org/debian-ports unstable/main riscv64 libgd3 riscv64 2.3.3-7
  404  Not Found [IP: 146.75.118.132 80]
Err:33 http://deb.debian.org/debian-ports unstable/main riscv64 libc-devtools riscv64 2.36-4
  404  Not Found [IP: 146.75.118.132 80]
Err:34 http://deb.debian.org/debian-ports unstable/main riscv64 linux-libc-dev riscv64 6.0.6-2
  404  Not Found [IP: 146.75.118.132 80]
Err:35 http://deb.debian.org/debian-ports unstable/main riscv64 libcrypt-dev riscv64 1:4.4.30-1
  404  Not Found [IP: 146.75.118.132 80]
Err:36 http://deb.debian.org/debian-ports unstable/main riscv64 rpcsvc-proto riscv64 1.4.2-4
  404  Not Found [IP: 146.75.118.132 80]
Err:37 http://deb.debian.org/debian-ports unstable/main riscv64 libc6-dev riscv64 2.36-4
  404  Not Found [IP: 146.75.118.132 80]
Err:38 http://deb.debian.org/debian-ports unstable/main riscv64 manpages-dev all 6.01-1
  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool/main/m/manpages/manpages_6.01-1_all.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/b/binutils/binutils-common_2.39-8_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/b/binutils/libbinutils_2.39-8_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/b/binutils/libctf-nobfd0_2.39-8_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/b/binutils/libctf0_2.39-8_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/b/binutils/binutils-riscv64-linux-gnu_2.39-8_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/b/binutils/binutils_2.39-8_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/m/mpfr4/libmpfr6_4.1.0-3_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/m/mpclib3/libmpc3_1.2.1-2_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-12/cpp-12_12.2.0-9_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-defaults/cpp_12.2.0-1_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool/main/f/fonts-dejavu/fonts-dejavu-core_2.37-2_all.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/f/fontconfig/fontconfig-config_2.13.1-4.5_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-12/libcc1-0_12.2.0-9_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-12/libgomp1_12.2.0-9_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-12/libasan8_12.2.0-9_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-12/libgcc-12-dev_12.2.0-9_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-12/gcc-12_12.2.0-9_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/gcc-defaults/gcc_12.2.0-1_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/b/brotli/libbrotli1_1.0.9-2%2bb4_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/glibc/libc-dev-bin_2.36-4_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/libp/libpng1.6/libpng16-16_1.6.38-2_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/f/freetype/libfreetype6_2.12.1%2bdfsg-3_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/f/fontconfig/libfontconfig1_2.13.1-4.5_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/libj/libjpeg-turbo/libjpeg62-turbo_2.1.2-1%2bb1_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/j/jbigkit/libjbig0_2.1-6_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/libw/libwebp/libwebp7_1.2.2-2%2bb1_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/t/tiff/libtiff5_4.4.0-5_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool/main/libx/libx11/libx11-data_1.8.1-2_all.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/libx/libx11/libx11-6_1.8.1-2_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/libx/libxpm/libxpm4_3.5.12-1_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/libg/libgd2/libgd3_2.3.3-7_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/glibc/libc-devtools_2.36-4_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/l/linux/linux-libc-dev_6.0.6-2_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/libx/libxcrypt/libcrypt-dev_4.4.30-1_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/r/rpcsvc-proto/rpcsvc-proto_1.4.2-4_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool-riscv64/main/g/glibc/libc6-dev_2.36-4_riscv64.deb  404  Not Found [IP: 146.75.118.132 80]
E: Failed to fetch http://deb.debian.org/debian-ports/pool/main/m/manpages/manpages-dev_6.01-1_all.deb  404  Not Found [IP: 146.75.118.132 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
root@debian:~# 

Could anyone be so kind as to explain what is going wrong?

Kind regards

Eberhard Binder

**Comment by eugene-tarassov** [comment, marked as answer]
Either you didn't run `sudo apt update` or the Debian website is down.

**Reply by EberhardBinder** [reply]
So what can I do to get the system into running?

Kind regards

Eberhard Binder
